generator/discovery/update_discovery_doc.sh

```diff
@@ -23,7 +23,10 @@ function print_service_textproto() {
   service_proto_path="${1#protos/}"
   product_path="${service_proto_path%/*.proto}"
   initial_copyright_year=$(date +"%Y")
-  cat <<_EOF_
+
+  (
```

Comment:
I am too lazy to figure out what these commands do. My guess is that it fills in the text between "Begin" and "End" with the new additions. But doesn't that get overwritten the next time we update the discovery doc?

Also do we care about sorting?

---

A solution where we have all the `rest_services {}` between "Begin" and "End" is close. The issue is with the copyright year which is variable. Ugh.

If we don't care about sorting, we can have one marker, and just insert the new `rest_services {}` above it in the config.